x/programs/rust/examples/automated-market-maker/src/tests.rs

```diff
@@ -0,0 +1,56 @@
+// #[cfg(test)]
```

Comment:
This is a todo and wont be done on this PR. Trying to get the amm merged before state schema changes.